helm/hpcc/templates/sasha.yaml

```diff
@@ -1,6 +1,10 @@
 {{ range $.Values.sasha -}}
 {{- if not .disabled -}}
-{{- $commonCtx := dict "root" $ "me" . }}
+{{- $secretsCategories := list "system" }}
+{{- if or (has "data" .access) -}}
```

Comment:
No or condition here.

Reply:
Test on "access" is too early, it isn't set at this point (it's set on line 10).